Geometry_Engine/Create/ShapeProfileCurves.cs

```diff
@@ -176,19 +180,32 @@ public static List<ICurve> BoxProfileCurves(double width, double height, double
 
         /***************************************************/
 
-        public static List<ICurve> FabricatedBoxProfileCurves(double width, double height, double webThickness, double topFlangeThickness, double botFlangeThickness)
+        public static List<ICurve> FabricatedBoxProfileCurves(double width, double height, double webThickness, double topFlangeThickness, double botFlangeThickness, double weldSize)
         {
             List<ICurve> box = RectangleProfileCurves(width, height, 0);
-            List<ICurve> innerBox = RectangleProfileCurves(width - 2 * webThickness, height - (topFlangeThickness + botFlangeThickness), 0);
-            double diff = botFlangeThickness- topFlangeThickness;
 
-            if (diff != 0)
-            {
-                Vector v = new Vector() { X = 0, Y = diff / 2, Z = 0 };
-                innerBox = innerBox.Select(x => Geometry.Modify.ITranslate(x, v)).ToList();
-            }
+            List<ICurve> welds = new List<ICurve>();
```

Comment:
Will need a check here somewhere if the weldsize is 0 or not. If it is zero, currently 0-length lines are added to the edges. 
![image](https://user-images.githubusercontent.com/22005920/70235618-9491b680-1763-11ea-9bce-eec9d692822f.png)

basically, if weld-size == 0, make sure it does the same thing as before, if not, what you have here seem fine!